User Guide/Projects.md

```diff
@@ -746,6 +769,14 @@ How to **publish** via **HTTP Post**:
 **Step 4:** Set msg as **Body**
 
 ![Image not Available](/assets/Fig104.png)
+
+Here are the steps to publish a topic using HTTP REST in ADEX application:
+
+- Identify the endpoint: You need to know the endpoint URL of the server that is configured to receive the message. This URL will typically include the protocol (e.g., http:// or https://), the hostname or IP address of the server, and a path to the resource that will receive the message.	
```

Comment:
"(e.g., http:// or https://)" -> "(e.g. https://)"
"the hostname or IP address of the server" -> "the hostname of the server"

Reply:
done